fbchat/_client.py

```diff
@@ -976,6 +976,42 @@ def getUserActiveStatus(self, user_id):
         """
         return self._buddylist.get(str(user_id))
 
+    def _fetchImages(self, thread_id):
+        data = {"id": thread_id, "first": 12}
```

Comment:
Does it make sense to implement some sort of incremental fetching amount? E.g. start by fetching the first 10 images, and the next time 20, and the next time 40, and so on?

Reply:
I am scared to do more than the messengers default 12. But yeah, I could do that.

Reply:
My bad! After digging around some more it looks like if you enter "big screen" mode to view images it fetches 48 of them. I'll do 48 items at the time.